src/components/BooksList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from './Book';
+
+const BooksList = () => {
```

Comment:
- Move this component to `src/containers`
The convention about the container/component pattern is that components with some logic have to placed in the container folder